hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -3054,6 +3054,13 @@ func deleteControlPlaneOperatorRBAC(ctx context.Context, c client.Client, rbacNa
 	return nil
 }
 
+func deleteClusterAPIClusterRoleBinding(ctx context.Context, c client.Client, controlPlaneNamespace string) error {
+	if _, err := deleteIfNeeded(ctx, c, &rbacv1.ClusterRoleBinding{ObjectMeta: metav1.ObjectMeta{Name: "cluster-api-master-" + controlPlaneNamespace}}); err != nil {
```

Comment:
where is this `cluster-api-master-` coming from? I'm not aware of any anywhere we set this prefix.